source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,268 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+   - Team and Channel management pages (but without channel moderation, e.g. read-only channels).
+ - Features moved from high SKU to mid SKU:
+   - SSO with OpenID Connect, SAML, Google and O365
+   - O365 integrations including MS Teams Calling and MS Calendar
+   - Jira multi-server support
+   - Advanced team permissions
+   - Read-only announcement channels
+ - Features added to high-end SKU:
+   - Shared channels
+   - Certificate-based authentication
+   - Granular data retention
+   - Granular audit logs w/export
+   - Custom admin roles
+ - E20, Professional, and Enterprise license SKUs are now supported for installing Enterprise plugins.
+
+#### Beta features Promoted to General Availability
+   - Archived channels
+   - Compliance exports
+   - Custom terms of service
+   - Guest accounts
+   - System roles
+   - Plugins
+
+#### Permalink Previews
+ - Added support for permalink previews for posts in Mattermost. Previews are generated to minimize context switching when sharing message links in Channels.
+
+#### Tutorial Updates
+ - Added a tip to the **Getting Started** page for downloading Desktop Apps.
+ - Updated tutorial icons and changed text content in tutorial tips.
+ - Updated the default treatment for the ``Add channel`` button to the current color and by team name.
+ - Added a tutorial tip for new settings and status buttons.
+ - Added a tip for the product switcher. This tip is skipped if not applicable.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added “Invite People” to the main "+" button below the hamburger menu.
+ - The whole category bounds are now highlighted while holding a channel above a category name on the left-hand side.
+ - Updated **Account Settings > Display > Timezone** to be more user friendly.
+ - New theme agnostic file preview modal takes up the full screen. The file preview now has information about the user, channel, and the file, and moves away from text-based buttons to icon-based buttons.
+ - Increased the limit of uploaded file attachments per post from 5 to 10.
+ - Added desktop notifications for followed Threads.
+ - Hungarian and English-Australian are now official languages.
+ - Added a **Start Trial** call-to action to the **Main Menu**.
+ - Changed H1-H3 heading font from Open Sans to Metropolis.
+
+#### Performance
+ - Improved typing performance when the emoji autocomplete is open.
+ - Added performance improvements for draft storage with multiple tabs open.
+ - Improved performance of draft loading.
+
+#### Integrations
+ - Dropped support for left-hand side-specific bot icons.
+ - Added a "rest field" to the App command parser.
+ - Added support for React components in channel header tooltips registered by plugins.
+ - Exported ``ChannelInviteModal`` and ``ChannelMembersModal`` components for plugins.
+ - Removed a deprecated "Backend" field from the plugin manifest.
+ - Converted the "Executables" field in the plugin manifest to a map.
```

Comment:
1., 5. and 6. are breaking changes and should be marked as such. Maybe moving them into `Plugin Changes` and changing the wording would make that more clear.

Reply:
Added to the "Important Upgrade Notes" section.